graphql/schemabuilder/input.go

```diff
@@ -230,7 +234,7 @@ func wrapWithZeroValue(inner *argParser) *argParser {
 			return inner.FromJSON(value, dest)
 		},
 		Type: inner.Type,
-	}
+	}, fieldArgTyp
```

Comment:
can we add an introspection test for this?

Reply:
Done: https://github.com/samsarahq/thunder/pull/219